### PR TITLE
Don’t forget luxury cars

### PR DESCRIPTION
**Pull request by Stupefacient**
Luxury cars and readme minor style correction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,10 +245,10 @@ There are 4 modes to select on the steering wheel and/or the onroad camera scree
...
 |    `hkg`     | Hyundai/Kia/Genesis(HKG)-specific branches | Only for HKG cars.                                                                                                              |
-|   `honda`    | Honda-specific branches                    | Only for Honda cars.                                                                                                            |
-|   `toyota`   | Toyota-specific branches                   | Only for Toyota cars.                                                                                                           |
-|   `subaru`   | Subaru-specific branches                   | Only for Subaru cars.                                                                                                           |
-| `volkswagen` | Volkswagen-specific branches               | Only for Volkswagen cars.                                                                                                       |
+|   `honda`    | Honda/Acura - Specific branches                    | Only for Honda cars.                                                                                                            |
```
Comment (Moodkiller) [inline README.md:248]: Only for Honda cars -> Only for Honda/Acura cars

File: README.md [hunk 2]
```diff
@@ -245,10 +245,10 @@ There are 4 modes to select on the steering wheel and/or the onroad camera scree
...
+|   `toyota`   | Toyota/Lexus - Specific branches                   | Only for Toyota cars.                                                                                                           |
```
Comment (Moodkiller) [inline README.md:249]: Only for Toyota cars -> Only for Toyota/Lexus cars

**Review by angels7777 — COMMENTED** [review]
Does this program mean that you are GPS tracking people on foot utilizing software on a car camera and computer

**Comment by sunnyhaibin** [comment]
Thanks!

**Pull request merged**